### ISSUE TITLE
Ren'Py Launcher Not Responding

### ISSUE THREAD

**Issue by Torvyn** [renpy/renpy#5846]
Hello!

I recently got back into Renpy. I downloaded the latest build, Renpy 8.3.2. It worked fine at first, but now, for some reason, the launcher will not work. It instantly goes into "not responding." I have tried reinstalling and installing it into a different folder path. Nothing has worked.

This issue started a bit after installing VS Code and its extensions for Renpy and Python, such as Pylance. However, I am not positive if its directly responsible for the issue, since after uninstalling it the issue stayed.

The log file says everything loaded fine, there is no traceback file.

Any help would be much appreciated!

**Comment by danac78** [comment]
Did you install Visual Code via the Launcher or manually? The reason I ask is because the Launcher allows you to download Visual Code with the current version of Renpy Extension (which had been changed recently). If you did not do it through the Launcher:

Try uninstalling Visual Code completely, restarting, and loading the Launcher again. If it works after that point, try going to Preferences in the Launcher, then Text Editor, and click on Visual Code. It will download and install both Visual Code and the current version of the Renpy extension.

Another thing you can try if uninstalling Visual Code does not fix it, try on another User account for your system. User Profiles (especially Windows) can cause strange behaviors if something happens (yes, that is vague, but there is a lot of things that occurs with a User Profile). If that fails and you have access to another system, check to see if the same behavior happens on that.

**Comment by danac78** [comment]
You can also try:

https://nightly.renpy.org/

Select a build from the Ren'Py 8 Fix column. These are nightly fixes that fix issues found in Renpy that might not be in 8.3.2 whereas Renpy 8 Development meant for developing the next version (in this case, 8.4) of Ren'Py. This way fixes could be released sooner without waiting for Renpy 8.4 to come out.:)

**Comment by juniorthejunior** [comment]
This sounds like https://github.com/renpy/renpy/issues/5325 - did you per chance move the Launcher's window to a secondary monitor, and close it there the last time that it was working properly?

If so, try creating a new .rpy file in your `renpy-8.3.2-sdk/launcher/game/` folder, and write the following inside it:
```py
init 999 python:
    preferences.restore_window_position = False
```
See if that indeed fixes the issue.

**Comment by Torvyn** [comment]
> This sounds like #5325 - did you per chance move the Launcher's window to a secondary monitor, and close it there the last time that it was working properly?
> 
> If so, try creating a new .rpy file in your `renpy-8.3.2-sdk/launcher/game/` folder, and write the following inside it:
> 
> ```python
> init 999 python:
>     preferences.restore_window_position = False
> ```
> 
> See if that indeed fixes the issue.

It worked, thank you so much!

**Comment by Torvyn** [comment]
Fixed it by adding a new file in renpy-8.3.2-sdk/launcher/game/

named the file fix.rpy, and adding the following code

init 999 python:
    preferences.restore_window_position = False
